driver/driver.go

```diff
@@ -229,3 +277,20 @@ func (r *rows) Next(dest []sqldriver.Value) error {
 func init() {
 	sql.Register("mysql", driver{})
 }
+
```

Comment:
please describe the usage as the comment. And maybe you want to return an error?

Reply:
Hi @atomicules , don't forget this comment. It also causes the CI failure

Reply:
Addressed (hopefully) in commit https://github.com/go-mysql-org/go-mysql/pull/673/commits/09ab3510215f59477cc56624c8d9e32785b547db